### ISSUE TITLE
In Safari 17.6 on macOS 13.6.9 the Design Selector Cards have an incorrect height value

### ISSUE THREAD

**Issue by DAnn2012** [impress-org/givewp#7518]
## Details

Hi, I would like to point out that, from some tests I have done, using GiveWP (3.15.1) and WordPress (6.6.1) in Safari Version 17.6 (18618.3.11.11.7, 18618) on macOS Ventura 13.6.9 (22G830), the Design Selector Cards have an incorrect height value. Therefore, the third card almost completely covers the button.

## Expected Behavior

The modal with the design selector cards should display like in Chrome on Windows 10 (which I tested and it is correct).

Changing the CSS like this seems to fix the problem:

```
.show-design-selector-modal .givewp-design-selector--card {
	height: auto;
}
```

Source code:

https://github.com/impress-org/givewp/blob/e1b19afead45a9f00b78c5928baf376436026014/src/FormBuilder/resources/js/form-builder/src/styles/_onboarding.scss#L121-L125

## Visuals

Before:
![givewp1](https://github.com/user-attachments/assets/5d687742-6b92-47b4-b2ba-547fa49f1ade)

After:
![givewp2](https://github.com/user-attachments/assets/5f5cc68d-aad9-42f4-96c1-d121801c4c60)

## System Information

<details><summary>Details</summary>

GiveWP 3.15.1
WordPress 6.6.1 (it_IT)
Safari Version 17.6 (18618.3.11.11.7, 18618)
macOS Ventura 13.6.9 (22G830)

</details>

**Comment by JoshuaHungDinh** [comment]
Hi @DAnn2012,

Thank you for bringing this to our attention. This looks like a straightforward fix. We've added it to our internal task list and will work on it shortly. We’ll update this issue as soon as the fix is implemented.

Thanks for your patience!

**Comment by JoshuaHungDinh** [comment]
Released: GiveWP 3.16.2
